crates/ironrdp-client/src/app.rs

```diff
@@ -56,25 +59,36 @@ impl App {
         })
     }
 
-    fn send_resize_event(&mut self) {
+    fn send_resize_event(&mut self) -> anyhow::Result<()> {
         let Some(size) = self.last_size.take() else {
-            return;
+            return Ok(());
         };
         let Some((window, _)) = self.window.as_mut() else {
-            return;
+            return Ok(());
         };
         let scale_factor = (window.scale_factor() * 100.0) as u32;
 
-        let _ = self.input_event_sender.send(RdpInputEvent::Resize {
-            width: u16::try_from(size.width).unwrap(),
-            height: u16::try_from(size.height).unwrap(),
-            scale_factor,
-            // TODO: it should be possible to get the physical size here, however winit doesn't make it straightforward.
-            // FreeRDP does it based on DPI reading grabbed via [`SDL_GetDisplayDPI`](https://wiki.libsdl.org/SDL2/SDL_GetDisplayDPI):
-            // https://github.com/FreeRDP/FreeRDP/blob/ba8cf8cf2158018fb7abbedb51ab245f369be813/client/SDL/sdl_monitor.cpp#L250-L262
-            // See also: https://github.com/rust-windowing/winit/issues/826
-            physical_size: None,
-        });
+        let width = u16::try_from(size.width).context("invalid `width`: out of range integral conversion")?;
```

Comment:
suggestion: Can stick to a non-faillible signature, and `expect`, a screen size bigger than 65535 × 65535 is not something we need to worry about for a while

Reply:
Done